Clicker/src/Clicker.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+export class ClickApp  extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            counter: 0
+        }
+        this.countPlus = this.countPlus.bind(this);
+        this.countMinus = this.countMinus.bind(this);
+        this.countReset =  this.countReset.bind(this);
+    }
+
+    //функции кнопок
+    countPlus(){
+        this.setState((prevstate) => ({
+            counter: prevstate.counter + 1
+        }))
+    }
+    countMinus(){
+        this.setState((prevstate) => ({
+            counter: prevstate.counter - 1
+        }))
+    }
+    countReset(){
+        this.setState((prevstate) => ({
```

Comment:
Do not need a callback function here: 
```suggestion
        this.setState({ counter: 0 })
```